src/plugins/api/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "api",
```

Comment:
What about "index-patterns"? We could then move the index pattern stuff out of `ui` and into this plugin.